src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -488,7 +493,7 @@ public boolean isQuestionFullyAnsweredByTeam(FeedbackQuestionAttributes question
                 }
             }
         }
-        return numberOfResponsesNeeded <= 0 ? true : false;
+        return numberOfResponsesNeeded <= 0;
```

Comment:
Just noticed that the meaning of `numberOfResponsesNeeded` in this method is different from that in the method above (ie. in `isQuestionFullyAnsweredByUser`). You might want to consider changing this method so that the meaning of this variable is consistent with that of the previous one.